dev-utils/src/commands/genesis_command.rs

```diff
@@ -22,8 +22,9 @@ use client_common::tendermint::types::{Genesis, Time};
 use client_common::{ErrorKind, Result, ResultExt};
 
 use crate::commands::genesis_dev_config::GenesisDevConfig;
+use client_common::gen_keypackage;
```

Comment:
perhaps no need to have that function in `client_common`, it can be moved to dev-utils?

Reply:
ok,
i'll also remove it from client-cli, because no longer necessary.
user can use dev-utils to create key-package.

Reply:
working on now

Reply:
removing "gen-keypackae" in client-cli, also client-common

Reply:
why is it used in client-rpc?

Reply:
yes, it's used in integration test.
can be replaced with dev-utils

Reply:
ok to be replaced then in the integration test?

Reply:
ok, i'll change integration test,
and remove from client-rpc.